src/Core/Components/List/FluentAutocomplete.razor

```diff
@@ -0,0 +1,112 @@
+                           disabled="@Disabled"
+                           placeholder="@Placeholder"
+                           aria-label="@(String.IsNullOrWhiteSpace(Title) ? Placeholder : Title)"
+                           aria-expanded="@(IsMultiSelectOpened ? "true" : "false")"
+                           aria-controls="@(IsMultiSelectOpened ? IdScroll : string.Empty)"
+                           current-value="@_valueText"
+                           @onclick="@OnDropDownExpandedAsync"
+                           @oninput="@InputHandlerAsync"
+                           @onkeydown="@KeyDownHandlerAsync"
+                           style="@ComponentWidth">
+            @* Selected Items *@
+            @if (this.SelectedOptions?.Any() == true)
+            {
+                <fluent-horizontal-scroll id="@IdScroll" style="width: 100%;" slot="start">
+                    <fluent-flipper onclick="document.getElementById('@IdScroll').scrollToPrevious()"
+                                    slot="previous-flipper"
+                                    aria-hidden="false"
+                                    role="button"
+                                    tabindex="0"
+                                    class="previous fluent-autocomplete-previous"
+                                    direction="previous"></fluent-flipper>
+                    <fluent-flipper onclick="document.getElementById('@IdScroll').scrollToNext()"
+                                    slot="next-flipper"
+                                    aria-hidden="false"
+                                    role="button"
+                                    tabindex="0"
+                                    class="next fluent-autocomplete-next"
+                                    direction="next"></fluent-flipper>
+                    @foreach (var item in this.SelectedOptions)
+                    {
+                        if (SelectedOptionTemplate == null)
+                        {
+                            <FluentBadge Appearance="@FluentUI.Appearance.Neutral"
+                                         OnDismissClick="@(e => RemoveSelectedItemAsync(item))"
+                                         aria-label="@GetOptionText(item)">
+                                @GetOptionText(item)
+                            </FluentBadge>
+                        }
+                        else
+                        {
+                            @SelectedOptionTemplate(item)
+                        }
+                    }
+                    &nbsp;
+                </fluent-horizontal-scroll>
+            }
+            <svg width="12" height="12" style="cursor: pointer;" xmlns="http://www.w3.org/2000/svg" slot="end" @onclick="@OnDropDownExpandedAsync">
+                <path d="M2.15 4.65c.2-.2.5-.2.7 0L6 7.79l3.15-3.14a.5.5 0 11.7.7l-3.5 3.5a.5.5 0 01-.7 0l-3.5-3.5a.5.5 0 010-.7z"></path>
+            </svg>
```

Comment:
Do we not want to use an Icon here? Or is it not available in the Fluent UI System Icons set

Reply:
Yes, that's the "Bottom arrow" icon at the end of the textfield, to open the popup.
May be better to include this code in `CoreIcon`. I will to that in a next PR.